AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/sample.json

```diff
@@ -16,7 +16,7 @@
 			"conda create --name user_tensorflow -y",
 			"conda activate user_tensorflow",
 			"conda install -c ${ONEAPI_ROOT}/conda_channel tensorflow numpy pyyaml scikit-learn schema lpot ilit matplotlib -y",
-		    "conda install runipy notebook -y"
+		        "conda install runipy notebook flatbuffers -y"
```

Comment:
same here. might be better to specify a specific conda env for installation.

Reply:
It's good suggestion. Current cmd stype is working well.
But I don't want to change more in the sample.json file, because current CI fault issue would be related with this file.
Less necessary change could help narrow down the scope.

I will update it according to the comment when this PR is passed.

Reply:
Yes, accept it.